src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -871,6 +863,30 @@ def get_mode(mode: str) -> "AccessorModes":
             return AccessorModes.GENERIC
 
 
+class MakeReadOnly:
+    """Removes 'set_state()' attribute to implement read-only behaviour."""
+
+    def __init__(self, cmd):
+        self._cmd = cmd
+        self._unwanted_attr = ["set_state", "setState"]
```

Comment:
Can we make this list a class attribute?

Reply:
Sure, Done.